tests/quasi-newton/parallel/TestQN6.cpp

```diff
@@ -0,0 +1,22 @@
+#ifndef PRECICE_NO_MPI
+
+#include "testing/Testing.hpp"
+
+#include <precice/precice.hpp>
+#include "../helpers.hpp"
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(QuasiNewton)
+BOOST_AUTO_TEST_SUITE(Parallel)
+BOOST_AUTO_TEST_CASE(TestQN6)
+{
+  PRECICE_TEST("SolverOne"_on(2_ranks), "SolverTwo"_on(2_ranks));
+  // serial coupling, IQN-IMVJ (which is identical to IQN-ILS as only first timestep is considered), relaxed QR2 filter
```

Comment:
Is this comment actually correct? The config looks like this test does 5 time windows?

Reply:
Yes, it's 5 time windows. I'll improve them. 
And I would like to add the goals of relevant tests here in the comments so we don't need to do a lot of renaming and still provide some insight.